code/modules/hydroponics/grown/misc.dm

```diff
@@ -1,18 +1,32 @@
-// Weeds
-/obj/item/seeds/weeds
-	name = "pack of weed seeds"
-	desc = "Yo mang, want some weeds?"
-	icon_state = "seed"
-	species = "weeds"
+// Starthistle
+/obj/item/seeds/starthistle
+	name = "pack of starthistle seeds"
+	desc = "A robust species of weed that often springs up in-between the cracks of spaceship parking lots"
+	icon_state = "seed-starthistle"
+	species = "starthistle"
 	plantname = "Starthistle"
-	lifespan = 100
+	lifespan = 70
 	endurance = 50 // damm pesky weeds
 	maturation = 5
 	production = 1
-	yield = -1
-	potency = -1
-	growthstages = 4
+	yield = 2
+	potency = 10
+	growthstages = 3
+	growing_icon = 'icons/obj/hydroponics/growing_flowers.dmi'
 	genes = list(/datum/plant_gene/trait/plant_type/weed_hardy)
+	mutatelist = list(/obj/item/seeds/harebell)
+
+/obj/item/seeds/starthistle/harvest(mob/user)
+	var/obj/machinery/hydroponics/parent = loc
+	var/seed_count = yield
+	if(prob(getYield() * 20))
+		seed_count++
```

Comment:
Why not just have `seed_count` be `yield + 1`, instead of this redundant increment? Also, since it's only used in this frame, perhaps it should be defined inside the if statement block